webapp/channels/src/utils/utils.tsx

```diff
@@ -1236,7 +1235,8 @@ export function getPasswordConfig(config: Partial<ClientConfig>) {
 }
 
 export function isValidPassword(password: string, passwordConfig: ReturnType<typeof getPasswordConfig>, intl?: IntlShape) {
-    let errorId = t('user.settings.security.passwordError');
+    // The translation strings used by this function are defined in admin_console/password_settings
+    let errorId = 'user.settings.security.passwordError';
```

Comment:
If it is defined in two places, can we export it and use the same message in both places?

Reply:
I can't really do that because this translation string is constructed by concatenation down below. I might do some cleanup of this in a followup PR, but I want that separate from this since I'd also want to move this out of `utils/utils` at the same time and change `getPasswordConfig` to a memoized selector to stop the components using it to re-render less